IdokladSdk/Models/ProformaInvoice/List/ProformaInvoiceListGetModel.cs

```diff
@@ -122,9 +123,9 @@ public class ProformaInvoiceListGetModel : IEntityId
         public bool IsIncomeTax { get; set; }
 
         /// <summary>
-        /// Gets or sets a value indicating whether is proforma a tax movement indication.
+        /// Gets a value indicating whether is proforma a tax movement indication.
         /// </summary>
-        public bool IsProformaTaxed { get; set; }
+        public bool IsProformaTaxed => IssuedTaxDocumentIds?.Any() ?? false;
```

Comment:
Asi bych toto chování bych nechal pouze v API, tady bych to nechal v původním tvaru { get; set; }. 
Kdyžtak se zeptej na názor Dejva.